patches/0002-Use-xdg-open-to-run-Steam-games-if-running-in-flatpa.patch

```diff
@@ -0,0 +1,77 @@
+From 6fabf3a0c04f9565e77e343feed53a9867f5a709 Mon Sep 17 00:00:00 2001
+From: gasinvein <gasinvein@gmail.com>
+Date: Mon, 25 Mar 2019 16:35:31 +0300
+Subject: [PATCH 2/2] Use xdg-open to run Steam games if running in flatpak
+
+---
+ lutris/runners/steam.py | 12 +++++++++++-
+ lutris/util/linux.py    |  7 +++++++
+ 2 files changed, 18 insertions(+), 1 deletion(-)
+
+diff --git a/lutris/runners/steam.py b/lutris/runners/steam.py
+index e0a1b1e4..05addd2d 100644
+--- a/lutris/runners/steam.py
++++ b/lutris/runners/steam.py
+@@ -188,6 +188,9 @@ class steam(Runner):
+                 return path[: -len("SteamApps")]
+ 
+     def get_executable(self):
++        if system.LINUX_SYSTEM.is_flatpak:
++            # Use xdg-open for Steam URIs in Flatpak
++            return system.find_executable("xdg-open")
```

Comment:
Use the `Gio` API, not spawning random commands.

Reply:
This would require rewriting a fairly large portion of the runner's code. Maybe it's better to open an issue/PR upstream?

Reply:
Yes it should be discussed upstream in the end.